executor/common_kvm_arm64.h

```diff
@@ -45,12 +47,13 @@ static volatile long syz_kvm_setup_cpu(volatile long a0, volatile long a1, volat
 	uint32 features = 0;
 	if (opt_count > 1)
 		opt_count = 1;
+	// Every set option is a bit in features[0].
 	for (uintptr_t i = 0; i < opt_count; i++) {
 		uint64 typ = opt_array_ptr[i].typ;
 		uint64 val = opt_array_ptr[i].val;
 		switch (typ) {
 		case 1:
-			features = val;
+			features |= (1 << val);
```

Comment:
It's better to make the fuzzer generate correct value with bits.
This looks like a bug in descriptions, the flags should contain (1<<val) values.
Then the fuzzer will be able to generate mask with multiple values.

Reply:
That's how we handle options on x86 and powerpc, where they do not necessarily correspond to bits.
We could probably just leave features as is for now, I don't see them actually affecting the coverage.

Reply:
Then at least add a TODO to the descriptions says that the current feature flags are wrong and should be 1<<val instead.

Reply:
Oh, wait, we actually are generating bits for the features:

```
kvm_setup_opt_arm64 [
# unions need at least 2 fields, but we have only 1 now, but we want to have it as union for future extention
        featur1 kvm_setup_opt_feature
        featur2 kvm_setup_opt_feature
]

kvm_setup_opt_feature {
        typ     const[1, int64]
        val     flags[kvm_vcpu_features_arm64, int64]
}
```

So it's fine to just drop this chunk from the patchset.

Reply:
I don't see this.

We have:
```
kvm_vcpu_features_arm64 = KVM_ARM_VCPU_POWER_OFF, KVM_ARM_VCPU_EL1_32BIT, KVM_ARM_VCPU_PSCI_0_2, KVM_ARM_VCPU_PMU_V3
```
and these are defined as follows:
https://elixir.bootlin.com/linux/latest/source/arch/arm64/include/uapi/asm/kvm.h#L105